app/move/index.js

```diff
@@ -32,6 +32,9 @@ router
   .route('/:moveId/cancel')
   .get(protectRoute('move:cancel'), cancel.get)
   .post(protectRoute('move:cancel'), cancel.post)
+router
+  .route('/:moveId/confirmation')
```

Comment:
Can probably use shorthand get:

```js
router.get('/:moveId/confirmation', protectRoute('move:create'), confirmation)
```

The above was using `.route()` because it was binding both `GET` and `POST`.